roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import search_journalctl
+import sys
+
+from time import time
+
+
+SAVED_TIMESTAMPS = list()
```

Comment:
I needed a way to retain a timestamp created by calling `time.time()` in order to [create a log entry](https://github.com/openshift/openshift-ansible/pull/4316/files#diff-a515f88fdca679286abb67d0bc994c98R188) that matched [an expected log entry set](https://github.com/openshift/openshift-ansible/pull/4316/files#diff-a515f88fdca679286abb67d0bc994c98R191).